src/test/java/raisetech/Student/management/Controller/StudentControllerTest.java

```diff
@@ -137,7 +137,7 @@ class StudentControllerTest {
 
   @Test
   void 受講生の例外処理が実行できて400で返ってくること() throws Exception {
-    mockMvc.perform(get("/exception"))
+    mockMvc.perform(get("/Find/{id}"))
```

Comment:
これまでの命名規則にし違うと最初の文字は小文字ですね。
アプリケーションに直接関係がないAPIなので間違ってても支障はないのですが気になったのでコメントです 🙏 

```suggestion
    mockMvc.perform(get("/find/{id}"))
```